nixos/modules/services/x11/display-managers/sddm.nix

```diff
@@ -39,7 +47,7 @@ let
     [Theme]
     Current=${cfg.theme}
     ThemeDir=/run/current-system/sw/share/sddm/themes
-    FacesDir=/run/current-system/sw/share/sddm/faces
+    FacesDir=${facesEnv}
```

Comment:
This is a breaking change. Instead, I think we should leave `FacesDir=/run/current-system/sw/share/sddm/faces` and just add `facesEnv` to `environment.systemPackages`.